src/wp-includes/wp-db.php

```diff
@@ -1402,6 +1403,10 @@ public function prepare( $query, ...$args ) {
 		}
 
 		foreach ( $args as $arg ) {
+			if ( is_a( $arg, WP_DB_Partial_Query::class ) ) {
```

Comment:
```suggestion
			if ( $arg instanceof WP_DB_Partial_Query ) {
```

`instanceof` is a more performant option than `is_a()`.
As this occurs within a loop, performance is affected exponentially.
When dealing with the database, any performance improvements are worth making.

See https://3v4l.org/93N1d